addons/advanced_fatigue/stringtable.xml

```diff
@@ -114,12 +114,14 @@
             <Japanese>手ぶれ因数</Japanese>
             <Chinesesimp>抖动因数</Chinesesimp>
             <Chinese>抖動因素</Chinese>
+            <German>Schwankungs Faktor</German>
```

Comment:
i used the suggested tabler tool so i have no idea what the ordering is, as tabler was suggested here i assumed it would be correct https://ace3mod.com/wiki/development/how-to-translate-ace3.html

I must have missed the other german translation PR when looking through the open PR'S, i though if #367 still lists them as missing they are missing but in this case just close the PR and use #6536